css/index.css

```diff
@@ -137,10 +175,103 @@ footer nav {
     justify-content: space-between;
     align-items: center;
     padding: 20px 2%;
-    font-size: 14px;
+    font-size: 1.4rem;
 }
 
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+@media (min-width: 1000px){
+	.middle-content .boxes .Box2 {
+	background: forestgreen;  
+}
+.middle-content .boxes .Box4{
+	background: darkorchid; }
+	.middle-content .boxes .Box6 {
+	background: hotpink; 
+	}
+.middle-content .boxes .Box8 {
+	background: indigo; 
+}
+.middle-content .boxes .Box10 {
+	background: lawngreen; 
+}
+}
+
+@media (max-width: 768px){
+	.container{
+
+		width: 100%; 
+        padding: 5px; 
+        margin: 0 auto;
+         
+    }
+    header {
+    	width: 100%; 
+    	display: flex; 
```

Comment:
The design-template that Josh provided has the nav and image as columns. You can see what happens when you implement flex-direction: columns. Then to make things look nice, align: center. You can get rid of display:flex an flex-stretch: stretch.